hooks/post/31-mongo_syspath.rb

```diff
@@ -0,0 +1,18 @@
+def install_syspaths
+  `rpm -qa | grep rh-mongodb34-syspaths`
+  if $?.success?
+    logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
+  else
+    logger.info 'rh-mongodb34-syspaths not present, installing.'
+    Kafo::Helpers.execute('yum install -y -q rh-mongodb34-syspaths')
+  end
+end
+
+# Fail if we are not using the Katello/Foreman Proxy Content scenario.
```

Comment:
Personally I think the code is self-documenting and could see this removed but it should at least match the code